automation/SetUp.ps1

```diff
@@ -0,0 +1,17 @@
+$SwitchName = "External VM Switch" 
+$ISOFile = "$HOME\Downloads\SERVER_EVAL_x64FRE_en-us-uni.iso"
+$VMName = 'minikube-m05' 
+$VM = @{
+    Name = $VMName;
+    MemoryStartupBytes = 1GB;
+    NewVHDPath = "${env:homepath}\.minikube\machines\$VMName\VHD.vhdx";
+    NewVHDSizeBytes = 10GB;
+    BootDevice = "VHD";
+    Path = "${env:homepath}\.minikube\machines\";
+    SwitchName = $SwitchName
+}
+New-VM @VM
+Set-VM -Name $VMName -ProcessorCount 2 -AutomaticCheckpointsEnabled $false
+Set-VMProcessor -VMName $VMName -ExposeVirtualizationExtensions $true
+Set-VMDvdDrive -VMName $VMName -Path $ISOFile
```

Comment:
I can't push changes to your fork but here's a suggestion that has worked for me to work around adding the answer file to the windows iso.

I've created a disk image that contains just the answer file and added a new dvd drive to the machine. Windows by default searches other disk drives for an unattended answer files meaning we can use default installation media and only update this iso if the answer file changes

`Add-VMDvdDrive -VMName $VMName -Path "$PSScriptRoot\auto-install.iso" -ControllerNumber 1 -ControllerLocation 1`

Reply:
See this PR: https://github.com/bobsira/MinikubeWindowsContainers/pull/1/files

Reply:
I've approved this PR and merged your changes.